newIDE/app/src/EventsSheet/index.js

```diff
@@ -1300,6 +1322,18 @@ export default class EventsSheet extends React.Component<Props, State> {
                               ),
                               click: this._openEventsContextAnalyzer,
                             },
+                            { type: 'separator' },
+                            {
+                              label: i18n._(t`Zoom In`),
+                              click: () => this.onZoomEvent('IN')(),
+                              accelerator: 'CmdOrCtrl+=',
+                            },
```

Comment:
Do you think we should have a max zoom level (open question)?

Reply:
I put a min level so that the user is not list if everything disappeared.
I can't think of a case where a max could be useful but humans find a way. I'll put one